src/style.css

```diff
@@ -1 +1,101 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: 400;
+  font-style: normal;
+}
+
+.body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.form {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  display: block;
+  margin-top: 20px;
+  outline: none;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.search_input_big {
+  font-size: 16px;
+  background: url(images/Search.svg) no-repeat;
+  background-size: 19px;
+  box-sizing: border-box;
+  background-position: 26px center;
+  padding-left: 62px;
+  height: 70px;
+  background-color: #ffff;
+}
+
+.search_input_small {
+  font-size: 14px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  background: url(images/Search.svg) no-repeat 13px center/11px 11px;
+  height: 42px;
+  padding-left: 33px;
+}
```

Comment:
Again, be cautious with setting fixed heights for containers. Ensure that the design allows for flexibility in content size to prevent overflow issues.